src/machine/machine_atsamd21.go

```diff
@@ -787,6 +787,38 @@ func getPMux(p uint8) sam.RegValue8 {
 		return sam.PORT.PMUX0_14
 	case 15:
 		return sam.PORT.PMUX0_15
+	case 16:
+		return sam.RegValue8(sam.PORT.PMUX1_0>>24) & 0xff
```

Comment:
The `& 0xff` isn't really necessary here: by converting to a `RegValue8` (actually `uint8`) you already have done the same operation.
On the other hand, it doesn't hurt and may be better for readability.

Reply:
That was why I left it there, for clarity.